examples/logical_enclosure.go

```diff
@@ -33,6 +32,16 @@ func main() {
 		fmt.Println(log_en)
 	}
 
+	// Update From Group
+	/*
```

Comment:
Why is this commented?

Reply:
This is when the update from group is needed. other times it cannot be called.